### PR TITLE
chart: support extra volumes/mounts evaluated as a template

### PR DESCRIPTION
**Pull request by no-reply**
in the case a particular application needs to add/mount extra volumes, these
values provide a hook for it to do so both on the rails application deployments,
and on the worker pods.

@samvera/hyrax-code-reviewers

### CODE REVIEW

File: chart/hyrax/min.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+ingress:
```
Comment (mcritchlow) [inline chart/hyrax/min.yaml:1]: Would it be worth referencing the use of this file in the chart's README?

Comment (no-reply) [reply chart/hyrax/min.yaml:1]: oh, we probably should just drop it.

this was for my local testing and got caught up in an overeager git add

Comment (no-reply) [reply chart/hyrax/min.yaml]: squashed out the extra `.yaml` files after porting the example to the inline docs.

**Review by dunn — APPROVED** [review]

File: chart/hyrax/min.yaml (hunk 1 above)
Comment (mcritchlow) [reply chart/hyrax/min.yaml]: LGTM!

**Pull request merged**